src/markdown.py

```diff
@@ -86,6 +86,7 @@ def md_parser(input_md: str, is_clean=True) -> Dict[str, str]:
     types = {}
     if is_clean:
         types = {"code_block", "link", "image", "code", "block_quote"}
+    global md_dict
```

Comment:
Can you explain a little while it's necessary to have this as a global variable?

Reply:
It isn't necessary anymore, my bad, it was just leftover from a previous idea we had for getting the data on another PR and has been removed now